spec/lib/expired_record_cleaner_spec.rb

```diff
@@ -0,0 +1,33 @@
+describe ExpiredRecordCleaner do
+  describe "fiscal year" do
```

Comment:
I am not sure I agree w the logic on making the method public so it can be tested manually -- if we are not confident in the behavior/loigic of a non-public method, we should either test its usage via more rigorous via automated tests for the necessary public methods or it is sufficiently complicated that we should break it out into its own class.

Did not see the deliveries specs before - noted!